lib/BannerPlugin.js

```diff
@@ -79,31 +79,13 @@ class BannerPlugin {
 							continue;
```

Comment:
These changes are sadly a breaking change for the plugin, but I can revert them since the need for this change is debatabale

Reply:
It's a major change anyway. It should be possible for the user to use the `path` helpers to extract the information from `filename`